db/migrate/20191122125659_add_hub_use_notification_to_certificate.rb

```diff
@@ -0,0 +1,6 @@
+class AddHubUseNotificationToCertificate < ActiveRecord::Migration[6.0]
+  def change
+    add_column :certificates, :hub_use_confirmation_at, :datetime, null: true
```

Comment:
```suggestion
    add_column :certificates, :certificate_in_use_at, :datetime, null: true
```
this way it will nicely link with `CertificateInUseEvent`

Reply:
or just **in_use_at**